paasta_tools/cli/utils.py

```diff
@@ -742,3 +744,34 @@ def pick_slave_from_status(status, host=None):
     else:
         slaves = status.marathon.slaves
         return slaves[0]
+
+
+def get_task_from_instance(cluster, service, instance, slave_hostname=None, task_id=None):
+    api = client.get_paasta_api_client(cluster=cluster)
+    if not api:
+        sys.exit(1)
+    try:
+        tasks = api.service.tasks_instance(service=service,
+                                           instance=instance,
+                                           slave_hostname=slave_hostname,
+                                           task_id=task_id).result()
+    except HTTPNotFound:
+        log.error("Cannot find instance {0}, for service {1}, in cluster {2}".format(instance,
+                                                                                     service,
+                                                                                     cluster))
+        sys.exit(1)
+    except HTTPError as e:
+        log.error("Problem with API call to find task details")
+        log.error(e.response.text)
+        sys.exit(1)
+    if not tasks:
+        log.error("Cannot find any tasks on host: {0} or with task_id: {1}".format(slave_hostname,
+                                                                                   task_id))
+        sys.exit(1)
+    task = tasks[0]
```

Comment:
You can't make guarantees about the ordering of this; is it really useful for the function just to return the first one? Do you have a usecase for 'given an appid, return one of its tasks (but I don't care which one)?'